src/python/WMCore/Storage/StageOutImpl.py

```diff
@@ -232,30 +235,62 @@ def __call__(self, protocol, inputPFN, targetPFN, options=None, checksums=None):
                 msg += "Error details:\n{}\n".format(str(ex))
                 logging.error(msg)
                 if retryCount == self.numRetries:
-                    #  //
+                    # //
                     # // last retry, propagate exception
                     # //
+                    logging.error("Maximum retries exhausted when trying to create the output directory")
                     raise ex
                 time.sleep(self.retryPause)
 
         # //
         # // Create the command to be used.
         # //
-        command = self.createStageOutCommand(sourcePFN, targetPFN, options, checksums)
-        #  //
+        # // This will actually only enforce the definition of the bearer token variables, still x509 auth activated as a backup
+        try:
+            command = self.createStageOutCommand(sourcePFN, targetPFN, options, checksums, authMethod="TOKEN")
+        except TypeError as ex:
```

Comment:
Is this block still relevant? If we updated all `createStageOutCommand` signature, then I think we can remove the try/except block, no?

Reply:
I prefer to keep a proper treatment of this error in the event it might happen.